### PR TITLE
First draft part 1 and 2

### PR DESCRIPTION
**Pull request by BarbzCodez**
- Cleaned the formatting a bit
- added parts 1 and 2, I think we need to add some more details

### CODE REVIEW

File: software-requirement-specification.md [hunk 1]
```diff
@@ -71,25 +79,37 @@ This subsection should:
 3. Specify the sources from which the references can be obtained.
 This information may be provided by reference to an appendix or to another document.
 
-## 2. General Description
+Which format is for the references (?)
```
Comment (CodyWallbridge) [inline software-requirement-specification.md:92]: is this intended to be left in?

Comment (CodyWallbridge) [reply software-requirement-specification.md:82]: lgtm otherwise

**Comment by CodyWallbridge** [comment]
lgtm otherwise

File: software-requirement-specification.md (hunk 1 above)
Comment (BarbzCodez) [reply software-requirement-specification.md:92]: No, thats a question I have for the TA and i dont wanna forget lol

**Pull request merged**